src/http/routes.js

```diff
@@ -88,5 +88,11 @@ router.put(
   hasVerifiedAccount,
   controllers.infusion.updateInfusion,
 );
+router.delete(
+  '/infusion/:infusionId',
+  authMiddleware,
+  hasUserPrivledge([HOSPITAL_ADMIN_USER.toLowerCase(), WARD_USER.toLowerCase()]),
+  controllers.infusion.deleteInfusion,
```

Comment:
You need to make use of the `hasConfirmedEmail` and `hasVerifiedAccount` middleware here

Reply:
okay

Reply:
fixed